### PR TITLE
DOCSP-27817 manual tags for compass repo

### PR DESCRIPTION
**Pull request by sarahemlin**
Hi @sarah-olson-mongodb & @mdb-ashley!  Here are the manual tags for the top 250 /docs-compass/ pages.  Please merge if everything looks good!

Thank you!
Sarah

## JIRA
https://jira.mongodb.org/browse/DOCSP-27817

### CODE REVIEW

File: source/import-export.txt [hunk 1]
```diff
@@ -6,6 +6,14 @@ Import and Export Data
 
 .. default-domain:: mongodb
 
+.. facet::
+   :type: programming_language
+   :values: json
```
Comment (mdb-ashley) [inline source/import-export.txt:11]: [question] Is JSON a programming language?

**Review by mdb-ashley — APPROVED** [review]
One question, but otherwise this LGTM! Thanks, Sarah!

File: source/import-export.txt (hunk 1 above)
Comment (sarahemlin) [reply source/import-export.txt:11]: JSON falls within the schema.org definition for programmingLanguage, so it's listed as one of our language values on the [Official Taxonomy](https://wiki.corp.mongodb.com/display/DE/Docs+Taxonomy?src=contextnavpagetreemode).

**Review by sarah-olson-mongodb — APPROVED** [review]
Thanks @sarahemlin! LGTM as well.

**Comment by sarahemlin** [comment]
@mdb-ashley & @sarah-olson-mongodb, I just put in a fix from DOP that updated the facet :type: to be :name: instead.  Please merge when you have a moment!

Thank you!
Sarah

**Pull request merged**